utilities/pipelines/resourceRemoval/Initialize-DeploymentRemoval.ps1

```diff
@@ -18,9 +18,9 @@ Optional. The name of the resource group the deployment was happening in. Releva
 Optional. The ID of the management group to fetch deployments from. Relevant for management-group level deployments.
 
 .EXAMPLE
-Initialize-DeploymentRemoval -DeploymentName 'virtualWans-20211204T1812029146Z' -TemplateFilePath "$home/ResourceModules/modules/network/virtual-wans/main.bicep" -resourceGroupName 'test-virtualWan-rg'
+Initialize-DeploymentRemoval -DeploymentName 'virtualWans-20211204T1812029146Z' -TemplateFilePath "$home/ResourceModules/modules/network/virtual-wan/main.bicep" -resourceGroupName 'test-virtualWan-rg'
```

Comment:
Are deployment names still in the form of 'camelCase-date'?

Reply:
Good catch. They will orientate themselves based on the path. This code actually is quite outdated now

```PowerShell
if ($templateFilePath -match '.*(\\|\/)Microsoft.+') {
    # If we can assume we're operating in a module structure, we can further fetch the provider namespace & resource type
    $shortPathElem = (($templateFilePath -split 'Microsoft\.')[1] -replace '\\', '/') -split '/' # e.g., AppConfiguration, configurationStores, .test, common, main.test.bicep
    $providerNamespace = $shortPathElem[0] # e.g., AppConfiguration
    $providerNamespaceShort = ($providerNamespace -creplace '[^A-Z]').ToLower() # e.g., ac

    $resourceType = $shortPathElem[1] # e.g., configurationStores
    $resourceTypeShort = ('{0}{1}' -f ($resourceType.ToLower())[0], ($resourceType -creplace '[^A-Z]')).ToLower() # e.g. cs

    $testFolderShort = Split-Path (Split-Path $templateFilePath -Parent) -Leaf  # e.g., common

    $deploymentNamePrefix = "$providerNamespaceShort-$resourceTypeShort-$testFolderShort" # e.g., ac-cs-common
}
```

will update it

Reply:
Fixed the name generation & updated the example. It should be `n-vw-t1-20211204T1812029146Z`